### ISSUE TITLE
put RC2 back

### ISSUE THREAD

**Issue by andyczerwonka** [apache/incubator-fury#1534]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-fury/issues) and found no similar issues.

### Version

RC2

### Component(s)

Java

### Minimal reproduce step

RC2 was removed from the snapshots repository, breaking builds that depend on it.

### What did you expect to see?

I don’t expect versions to disappear so quickly, I expect new ones to appear and only removing old ones after a deprecation cycle.

### What did you see instead?

RC2 was removed.

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by chaokunyang** [comment]
Hi, @andyczerwonka , sorry for removing rc2 jars, we're releasing rc3. I will add rc2 back and remove it when we have a final release.

**Comment by chaokunyang** [comment]
@andyczerwonka At the same time, could you try rc3 out?

**Comment by andyczerwonka** [comment]
> and remove it when we have a final release.

Let's not remove it until people have time to move off of it.

**Comment by andyczerwonka** [comment]
> could you try rc3 out?

Yes, but we'll do that locally first. Right now we need RC2 back to get build working.

**Comment by chaokunyang** [comment]
@andyczerwonka It's reuploaded, please check it . Please keep an eye on this, we will make a final release in next days, and remove thse release candicate.

**Comment by andyczerwonka** [comment]
**Please do not remove the RC versions**. Just add the final release when it's ready. This allows people to move onto the release at their own pace.
